Scripts/Python/Elonin.py

```diff
@@ -0,0 +1,8 @@
+from Plasma import *
+from PlasmaTypes import *
+
+class Elonin(ptResponder):
+    def __init__(self):
+        ptResponder.__init__(self)
+        self.id = -1
```

Comment:
Is it expected that this `id` is -1? All the other Age python files that I've found have a positive (usually in the thousands or more) `id` value...

Reply:
We'll want to manually assign an ID to this. Korman automatically exports this file with `self.id = -1` to ensure that AgeSDL works correctly.

Reply:
Should the ID be the same as the sequence prefix or something else?

Reply:
Any random 32-bit integer that doesn't cause the scripts review to detect a conflict will work.